procedures/form-new-working-group.md

```diff
@@ -0,0 +1,83 @@
+# Forming a new working group
+
+So, you would like to form a working group?
+
+## Step 1. Propose the working group
+
+**TBD** -- ironically, we haven't *quite* decided what the procedure
+is for this!  For now, a good idea is to hop on to #t-compiler on [the
+Zulip][cp] and open up a new topic to discuss your idea. Or,
+alternatively, join a meeting and leave a few comments.
+
+[cp]: ../about/chat-platform.md
+
+A good goal for this process is to try and identify a small number of
+**achievable deliverables**. It will be easiest to manage a working
+group if you have relatively clear, finite goals. Don't worry if you
+seem to be tackling an "infinite" problem space -- set some initial
+goals within that problem space. If, after you achieve those, you wish
+to continue, you can always add fresh goals or (maybe better) form a
+new working group.
+
+It's also good to identify an initial set of **group leads** -- the
+leads are the people who are driving the project forward overall. It's
+best if you have 2 or 3 leads who are exciting about the project,
+including at least one person from the compiler team, but there aren't
+strict rules here. See below for a few more details.
+
+## Step 2. Figure out key information
+
+You need to decide the following:
+
+**Who are the leads?** The leads are the people who are ultimately
+responsible for ensuring that the working group is making progress and
+for reporting on that progress to the compiler team and community as a
+whole. Basically, they are the people who are **driving** the project
+forward.  **The set of leads can also change over time.** Often, as
+new people join, they may start to play a leadership role, which is
+great!
+
+Leads often do the organizational work for the WG:
+
+- scheduling meetings
+- summarizing progress from the meetings into reports
+- reporting to the compiler team
+
+but these jobs can (and should, where possible) be delegated amongst
+the team as well.
+
+When possible, we prefer to have 2 or 3 leads per WG. 1 lead is ok,
+but not recommended, as it can be stressful. While not strictly
+required, typically at least one lead should be a full member of the
+compiler team.
+
+**Will you have a regular meeting?** If so, when? You can always
+change this later, of course. Many working groups opt for a weekly
+meeting held on Zulip.
```

Comment:
Is it worth mentioning that checking-in with the compiler team - so that the team knows the working group is still active and working away - is a requirement? Is that a requirement?

Reply:
That is definitely a requirement. It's not really something you need to "figure out". Maybe we need a kind of "step 0 -- what is a working group" section to talk about that?

To my mind, a working group is basically appropriate for any project that requires design and debate spanning any PR. So it is basically "something the compiler team is doing", and part of that is that we are going to check up on it from time to time =)

I suppose that a requirement of the leads is that at least one of them should be able to attend the meeting.

Reply:
I guess I'm wondering the extent to which a *"this is what you're signing yourself up for"* section is required. I think the few concerns I've got are mostly with the to-be-decided contents of this section:

> **TBD** -- ironically, we haven't *quite* decided what the procedure
is for this!  For now, a good idea is to hop on to #t-compiler on [the
Zulip][cp] and open up a new topic to discuss your idea. Or,
alternatively, join a meeting and leave a few comments.

I think I'm just cautious of it appearing as if the bar to starting a working group is low, until the working group/team has decided what that bar should be. 🤷‍♂️